calico_containers/calico_ctl/container.py

```diff
@@ -114,18 +115,6 @@ def container(arguments):
     validate_arguments(arguments)
 
     try:
-        workload_id = None
-        if "<CONTAINER>" in arguments:
-            container_id = arguments.get("<CONTAINER>")
-            if container_id.startswith("/") and os.path.exists(container_id):
-                # The ID is a path. Don't do any docker lookups
-                workload_id = escape_etcd(container_id)
-                orchestrator_id = NAMESPACE_ORCHESTRATOR_ID
-            else:
-                info = get_container_info_or_exit(container_id)
-                workload_id = info["Id"]
-                orchestrator_id = DOCKER_ORCHESTRATOR_ID
-
         if arguments.get("ip"):
```

Comment:
yup i mean the IP commands.  just nice to commonize the code when we get the chance.

i'm not too fussed if you want to just get this in.